Tests/KituraTests/TestStaticFileServer.swift

```diff
@@ -221,4 +221,8 @@ class TestStaticFileServer: XCTestCase {
         runGetResponseTest(path: "/@@Kitura-router@@/missing.file", expectedStatusCode: HTTPStatusCode.notFound)
     }
 
+    func testAbsolutePathFunction() {
```

Comment:
@joyychang This is late as this is already merged, and it's not really a big deal for this one, but in general we should add any new tests to `static var allTests` so they are tested on linux builds as well.